resource-sharing-partners/src/main/java/no/sikt/rsp/ResourceSharingPartnerHandler.java

```diff
@@ -12,25 +11,35 @@
 import java.net.http.HttpResponse;
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Collection;
 import java.util.List;
 import java.util.stream.Collectors;
 import no.nb.basebibliotek.generated.BaseBibliotek;
 import no.sikt.alma.generated.Partner;
 import no.unit.nva.s3.S3Driver;
 import nva.commons.core.Environment;
 import nva.commons.core.JacocoGenerated;
+import nva.commons.core.StringUtils;
 import nva.commons.core.paths.UnixPath;
 import nva.commons.core.paths.UriWrapper;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import software.amazon.awssdk.services.s3.S3Client;
 
+@SuppressWarnings("PMD.DataflowAnomalyAnalysis")
```

Comment:
Note to self: flytt denne

Reply:
Denne hadde jeg også før. Den er irriterende. Kasnkje vi burde fjerne den allerede i forkant en lurt plass?

Reply:
Ordnet